extensions/ql-vscode/src/codeql-cli/cli.ts

```diff
@@ -408,14 +408,16 @@ export class CodeQLCliServer implements Disposable {
       } catch (err) {
         // Kill the process if it isn't already dead.
         this.killProcessIfRunning();
-        // Report the error (if there is a stderr then use that otherwise just report the error cod or nodejs error)
+        // Report the error (if there is a stderr then use that otherwise just report the error code or nodejs error)
         const newError =
           stderrBuffers.length === 0
-            ? new Error(`${description} failed: ${err}`)
+            ? new Error(
+                `${description} failed with args ${argsString}:${EOL}${err}`,
```

Comment:
Thanks. I think this is good. Can I ask for one more thing? When I am debugging, often it is the list of args that I am most interested in, and I copy and paste it so I can run it directly in a terminal. Could you make the text more prominent? Eg- indented and on its own line?

Reply:
Sure, that sounds like it'll be helpful! How's this?

![screenshot of failed `resolveQueries` command](https://github.com/github/vscode-codeql/assets/42641846/a3bd19e7-0c5b-4201-b66a-023ceb44312e)